cht-config/app_settings.json

```diff
@@ -0,0 +1,584 @@
+          "translation_key": "targets.anc.active_pregnancies.title",
+          "icon": "icon-pregnancy",
+          "subtitle_translation_key": "targets.all_time.subtitle"
+        },
+        {
+          "id": "active-pregnancies-1+-visits",
+          "type": "count",
+          "goal": -1,
+          "translation_key": "targets.anc.active_pregnancies_1p_visits.title",
+          "icon": "icon-clinic",
+          "subtitle_translation_key": "targets.all_time.subtitle"
+        },
+        {
+          "id": "facility-deliveries",
+          "type": "percent",
+          "goal": -1,
+          "translation_key": "targets.anc.facility_deliveries.title",
+          "icon": "icon-mother-child",
+          "subtitle_translation_key": "targets.all_time.subtitle",
+          "dhis": {
+            "dataElement": "e22tIwy1nKR",
+            "categoryOptionCombo": "HllvX50cXC0",
+            "attributeOptionCombo": "HllvX50cXC0"
+          }
+        },
+        {
+          "id": "active-pregnancies-4+-visits",
+          "type": "count",
+          "goal": -1,
+          "translation_key": "targets.anc.active_pregnancies_4p_visits.title",
+          "icon": "icon-clinic",
+          "subtitle_translation_key": "targets.all_time.subtitle"
+        },
+        {
+          "id": "active-pregnancies-8+-contacts",
+          "type": "count",
+          "goal": -1,
+          "translation_key": "targets.anc.active_pregnancies_8p_contacts.title",
+          "icon": "icon-follow-up",
+          "subtitle_translation_key": "targets.all_time.subtitle"
+        }
+      ]
+    }
+  },
+  "create_user_for_contacts": {
+    "replace_forms": [
+      "replace_user"
+    ]
+  },
+  "transitions": {
+    "accept_patient_reports": false,
+    "conditional_alerts": false,
+    "default_responses": false,
+    "update_sent_by": false,
+    "registration": false,
+    "update_clinics": false,
+    "update_notifications": false,
+    "update_scheduled_reports": false,
+    "generate_patient_id_on_people": true,
+    "death_reporting": true,
+    "create_user_for_contacts": false,
+    "mark_for_outbound": true
+  },
+  "outbound": {
+    "FHIR_patient": {
+      "relevant_to": "doc.type === 'person' && doc.role == 'patient'",
+      "destination": {
+        "base_url": "http://localhost:5001",
+        "path": "/fhir/Patient",
+        "auth": {
+          "type": "basic",
+          "username": "interop-client",
+          "password_key": "openhim1"
+        }
+      },
+      "mapping": {
+        "resourceType": {
+          "expr": "'Patient'"
+        },
+        "gender": "doc.sex",
+        "name": {
+          "expr": "[ { \"use\":\"official\", \"family\": doc.name , \"given\": [ doc.short_name ] } ]",
+          "optional": false
+        },
+        "identifier.use": {
+          "expr": "'usual'"
+        },
+        "identifier.value": {
+          "expr": "'cht/'.concat(doc._id)"
+        },
+        "birthDate": "doc.date_of_birth",
```

Comment:
We are expecting `date_of_birth` not `birthDate`. We had a discussion yesterday about data transformation taking place on the mediator and not cht-core.